### PR TITLE
HQPorner

### PR DESCRIPTION
**Pull request by DogmaDragon**
Decided to start learning xPath and noticed that there was a request https://github.com/stashapp/CommunityScrapers/issues/73#issuecomment-1076214178 for this one, so here it is, a little barebone but it works. 

Couldn't figure out how to extract the image from the iframe. 

Feedback is welcome!

### CODE REVIEW

**Comment by bnkai** [comment]
I updated the last updated date to pass the validation and added the image part (tested only for a couple of scenes)

To get the image we need to use a `subscraper` and get the poster from the script text

**Comment by DogmaDragon** [comment]
Tested getting the image it works well with the latest player they use, but not with a different one. 
E.g. https://hqporner.com/hdporn/77657-Kink_-_Petite_Whore_Inserts_Her_Small_Hand_In_A_Butt_-_Everything_Butt.html
From what I understand it's because it doesn't pass the poster via script it just adds it in the <video poster="">. 

Not sure how much of a problem this is as it seems that only a minority of older videos use that player. 

Thank you for the information and example.

File: scrapers/HQPorner.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            - map:
+                HQPORNER: HQPorner
+      Image:
+        selector: //iframe[@width="560"]/@src
+        postProcess:
+          - replace: &https
+              - regex: ^//
+                with: "https://"
+          - subScraper:
+              selector: //script[contains(.,"poster=")]
```
Comment (bnkai) [inline scrapers/HQPorner.yml:35]:
```yaml
              selector: //video/@poster|//script[contains(.,"poster=")]
```
This should cover the `older` player case then. @DogmaDragon  can you verify for the scenes you mention?

Comment (DogmaDragon) [reply scrapers/HQPorner.yml:35]: Tested and it works.

**Pull request merged**